### PR TITLE
Fix/add radiant v2 base

### PR DESCRIPTION
**Pull request by LuPoYi**
- [x] update dependencies
- [x] add radiant v2 on base config

### CODE REVIEW

File: packages/lending/src/protocols/radiant-v2/configs.ts [hunk 1]
```diff
@@ -29,6 +29,13 @@ export const configs: Config[] = [
       PriceOracle: '0x0BB5c1Bc173b207cBf47CDf013617087776F3782',
     },
   },
+  {
+    chainId: common.ChainId.base,
+    contractMap: {
+      ProtocolDataProvider: '0x07d2DC09A1CbDD01e5f6Ca984b060A3Ff31b9EAF',
+      PriceOracle: '0xe373749cd9b2D379f7f6Dd595e5164498b922164',
```
Comment (LuPoYi) [inline packages/lending/src/protocols/radiant-v2/configs.ts:36]: https://docs.radiant.capital/radiant/contracts-and-security/base-contracts#miscellaneous

File: packages/lending/src/protocols/radiant-v2/configs.ts [hunk 2]
```diff
@@ -29,6 +29,13 @@ export const configs: Config[] = [
       PriceOracle: '0x0BB5c1Bc173b207cBf47CDf013617087776F3782',
     },
   },
+  {
+    chainId: common.ChainId.base,
```
Comment (Jeff-CCH) [inline packages/lending/src/protocols/radiant-v2/configs.ts:33]: please test getPortfolio on the new added base chain in lending-protocol.test.ts

Comment (LuPoYi) [reply packages/lending/src/protocols/radiant-v2/configs.ts:33]: added

**Review by Jeff-CCH — APPROVED** [review]
LGTM

**Pull request merged**